web3js-ext/src/web3/klaytn_tx.ts

```diff
@@ -116,9 +123,10 @@ export class KlaytnTx extends LegacyTransaction {
     this._klaytnType = toNumber(txData.type) as number;
     this.from = txData.from;
     this.chainId = txData.chainId;
+    // @ts-ignore
+    this.key = txData.key;
```

Comment:
Define 'key' in KlaytnTxData type at L10, then remove ts-ignore.

Reply:
Yes I did https://github.com/klaytn/web3klaytn/pull/507/commits/a45019917a79438eeefc27f034697c8daef05bfd